build/Makefile

```diff
@@ -85,6 +88,10 @@ include ./includes/$(osinclude)
 # personal includes, excluded from the git repository
 -include ./local-includes/*.mk
 
+ifdef DOCKER_RUN
```

Comment:
So I'm curious - what does this do, and why is it necessary?

Reply:
I couldn't get a our `docker run` type of target get the gcloud service account of the cloud builder to access our kubernetes cluster, so I ended up using a custom 2e2  image which is a `FROM gcr.io/cloud-builders/gcloud`  and has a gcloud already configured automagically.

Since I have run without docker in the build system, I need to call the make target without doing a docker run and calling the go test command straight. **So this line make sure I don't run `ensure-build-image` if DOCKER_RUN is empty, which otherwise would fail.**

I wanted to keep the runner using our make targets (install and test-2e2) so we don't end up with too many targets. Another option would be to have special targets only for the runner and we would replicate the `go test` and the `helm install` command. 

I'm ok to shift toward a multiple makefile system if you think that is clearer, let me know.

Reply:
SGTM. I (think) I follow.

I can't think of a better way to do this :smile: so yeah, let's run with it for now :+1: